src/pages/user/User.tsx

```diff
@@ -39,24 +40,19 @@ const StyledPrimaryUserInfo = styled.div`
 
 const User: React.FC = () => {
   const { t } = useTranslation('profile');
+  const user = useSelector((state: RootStore) => state.user);
+  const location = useLocation();
   const dispatch = useDispatch();
   const history = useHistory();
-  const location = useLocation();
 
   useEffect(() => {
-    const query = new URLSearchParams(location.search);
-    const orcidCode = query.get('code') || '';
-    const error = query.get('error') || '';
-    if (error) {
-      dispatch(orcidSignInFailure(t('feedback:error.orcid_login')));
-      history.push('/user');
-    } else if (orcidCode) {
-      dispatch(getOrcidInfo(orcidCode));
+    const hash = queryString.parse(location.hash);
```

Comment:
trenger vi en ny pakke bare for dette? Kan vi ikke bare fjerne første tegnet (#) for å få til det samme?

Reply:
tja, vi kan vel implementere det selv også, men vi får ikke tilbake /#access_token=12345,

dette er det vi får tilbake:
```
http://localhost:3000/user#access_token=34ed981d-b930-4c5d-bab7-b7febfe8c606&token_type=bearer&expires_in=599&tokenVersion=1&persistent=true&id_token=eyJraWQiOiJzYW5kYm94LW9yY2lkLW9yZy0zaHBnb3NsM2I2bGFwZW5oMWV3c2dkb2IzZmF3ZXBvaiIsImFsZyI6IlJTMjU2In0.eyJhdF9oYXNoIjoiYnF3OWt1NEtraXBZVVZ5T0k1VEc3ZyIsImF1ZCI6IkFQUC1RSEJKU1U5OVNaVjFMTVFKIiwic3ViIjoiMDAwMC0wMDAxLTUyMTUtNzA3NSIsImF1dGhfdGltZSI6MTU3NTUzMzA0MiwiaXNzIjoiaHR0cHM6XC9cL3NhbmRib3gub3JjaWQub3JnIiwiZXhwIjoxNTc1NTMzNzcxLCJnaXZlbl9uYW1lIjoiU2FyYWgiLCJpYXQiOjE1NzU1MzMxNzEsImp0aSI6ImE3YjUzNmFmLTQzZTMtNDk3Zi05ZWQwLTA4N2FkYzkyYTYxMiJ9.cBSOYdtU94SXuvUgVYvZyD-4rQIV98eImR-AQa8uuOEhcJqzIjdQPVRMtQV4bHaUsKuIdKu901BcRs1RxUwi6tSPpdW4hbXkWNxAQ0Xow73ndEkOSDRSC9qo1eP35L0LHvNBHsCzeiF1jRumSXOFi_ALmaQ7M_QhpjslT4nlYGjd0MZOG3VubKNdgLAN2Zus1cTfjqp2iMML6_wyEaEfLCitO7NywZGW1vA6Ym4MWXqdwzLm4VvclN6rKA40Vh6pXro7L4SQH9NeV2lHqeydedEgQkq7fx3UM8HdzU_bxEg_xVDsISDRt7m6Nu_OgK5_a9J4QVSYY9z4-QHjJpNTqw&tokenId=15774129
```

Reply:
aah. Syns egentlig react-router burde ha en eller annen hook for å få til det samme uten å dra inn enda en avhengighet... Men da får vi bare leve med det :)

Reply:
kan sjekke det ut litt mer 👍